deploy/kubernetes/README.md

```diff
@@ -4,7 +4,17 @@ cAdvisor uses [Kustomize](https://github.com/kubernetes-sigs/kustomize) to manag
 
 ## Usage
 
-From the cadvisor/ directory, to generate the base daemonset:
+Pick a [cAdvisor release](https://github.com/google/cadvisor/releases)
+```
+VERSION=v0.42.0
+```
+
+From the `cadvisor/deploy/kubernetes/base` directory, to update the image version([reference](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/image.md)):
```

Comment:
`cadvisor/deploy/kubernetes/base` -> `deploy/kubernetes/base/` because cadvisor dir is root.